program/plugins/nikto_domino.plugin

```diff
@@ -54,27 +54,28 @@ sub nikto_domino {
 
     # quit if banner is not from IBM/Lotus Domino
     if (($mark->{'banner'} !~ /lotus-?domino/i) && ($response->{'server'} !~ /lotus-?domino/i) && ( $res ne "200" && $content !~ /src="\/domcfg.nsf\// ) ) {
-	$is_domino=0;
-	}
+        $is_domino=0;
+    }
     else {
```

Comment:
@sullo Was this really intended in https://github.com/sullo/nikto/commit/560073603a4a4f4868e948a7bc94f620801045ea

Currently Domino is only detected if a banner exists OR domcfg.nsf exists AND one of the /admin.nsf, /admin4.nsf, admin5.nss, /webadmin.nsf.

In my current audit the domino server wasn't detected because the domcfg.nsf existed but none of the other .nsf files.